src/shared/components/entry/TaxonomyView.tsx

```diff
@@ -9,17 +9,26 @@ import { getEntryPath } from '../../../app/config/urls';
 
 import { Namespace } from '../../types/namespaces';
 
-import { OrganismData } from '../../../uniprotkb/adapters/namesAndTaxonomyConverter';
 import UniProtKBEvidenceTag from '../../../uniprotkb/components/protein-data-views/UniProtKBEvidenceTag';
-import OrganismDataView from '../views/OrganismDataView';
+import TaxonomyLightView from '../views/TaxonomyView';
+import {
+  isLineage,
+  Lineage,
+  TaxonomyDatum,
+} from '../../../supporting-data/taxonomy/adapters/taxonomyConverter';
 
 type TaxonomyDataProps = {
-  data: OrganismData;
+  data: TaxonomyDatum;
 };
 
-export const TaxonomyLineage: FC<{ lineage: string[] }> = ({ lineage }) => (
-  <>{lineage.join(' > ')}</>
-);
+export const TaxonomyLineage: FC<{ lineage: Lineage | string[] }> = ({
+  lineage,
+}) => {
+  if (isLineage(lineage)) {
+    return <></>;
```

Comment:
This should probably just be `null`
```suggestion
    return null;
```